### PR TITLE
Fix OSX docs tests.

### PR DESCRIPTION
**Pull request by mantid-roman**
Use the ellipsis in cases numbers may change due to round-off errors.

### CODE REVIEW

File: docs/source/fitting/fitfunctions/LatticeFunction.rst [hunk 1]
```diff
@@ -50,19 +50,18 @@ used to generate Bragg reflections that are expected for the crystal structure o
 
     a_true = 5.4311946
     a = np.round(parameters.cell(0, 1), 7)
-    a_err = np.round(parameters.cell(0, 2), 16)
+    a_err = np.round(parameters.cell(0, 2), 7)
 
     print("Refined lattice parameter: a = {0} +/- {1}".format(a, a_err))
     print("Difference from expected value: a_observed - a_expected = {0}".format(np.round(a - a_true, 7)))
-    print("Is this difference within the standard deviation? {0}".format("Yes" if np.fabs(a - a_true) <= a_err else "No"))
```
Comment (martyngigg) [inline docs/source/fitting/fitfunctions/LatticeFunction.rst]: Is there no way to get this to match?

File: docs/source/concepts/CrystalStructureAndReflections.rst [hunk 2]
```diff
@@ -253,7 +253,7 @@ to the process for :math:`F^2`:
 
     # Make list of tuples and sort by d-values, descending, include point group for multiplicity.
     reflections = sorted([(hkl, d, fsq, len(pg.getEquivalents(hkl))) for hkl, d, fsq in zip(hkls, dValues, fSquared)],
-                                    key=lambda x: x[1], reverse=True)
+                                    key=lambda x: x[1] - x[0][0]*1e-6, reverse=True)
```
[the remark this answers is not in the document]
Comment (mantid-roman) [reply docs/source/concepts/CrystalStructureAndReflections.rst]: In some cases two of the `x[1]` are almost equal and the difference is due to some "round-off" effects or particulars of the method that calculates them. So their lines can swap places in the sorted list. Adding `x[0][0]` to the 6-th digit makes sure that the order stays the same regardless of the exact values of `x[1]`

[the remark this answers is not in the document]
Comment (martyngigg) [reply docs/source/concepts/CrystalStructureAndReflections.rst]: I see. Thanks for the explanation.

**Review by martyngigg — APPROVED** [review]
This should make these tests more stable in the future.

File: docs/source/fitting/fitfunctions/LatticeFunction.rst (hunk 1 above)
Comment (mantid-roman) [reply docs/source/fitting/fitfunctions/LatticeFunction.rst]: There is no noise in the fitted data which could give the standard deviation one or two solid significant digits. We could manually set the limit on it about 1e-15 - 1e-14 which is the accuracy of double precision calculations. But I don't think it is useful.

**Pull request merged**